### PR TITLE
[patch] [mascore-1684] Update deprovision tasks pipelines

### PR DESCRIPTION
**Pull request by padmankosalaram**
As part of https://jsw.ibm.com/browse/MASCORE-1684?filter=-1

Pipelines changes expected on SaaS env to 

    deprovision-mas-app
    deprovision-mas-instance
    deprovision-mas-cluster (the cluster level resources only rather than the cluster itself)

This fix is only concerning Esstentials Inspection (MAS core and MVI only) 
So ignored db2, cp4d and kafka from these deprovision tasks for now.

### CODE REVIEW

**Review by racree — APPROVED** [review]

File: tekton/src/tasks/gitops/gitops-delete-kafka-config.yml.j2 [hunk 1]
```diff
@@ -55,10 +58,18 @@ spec:
         value: $(params.github_pat)
       - name: SM_AWS_REGION
         value: $(params.avp_aws_secret_region)
+      - name: OCP_SERVER
+        value: $(params.cluster_url)
```
Comment (andrercm) [inline tekton/src/tasks/gitops/gitops-delete-kafka-config.yml.j2:62]: why we're using `$(params.cluster_url)` to set `OCP_SERVER`?
should this be setting `CLUSTER_URL` or the other way around?

File: tekton/src/tasks/gitops/gitops-deprovision-app-config.yml.j2 [hunk 2]
```diff
@@ -60,10 +63,18 @@ spec:
         value: $(params.force_removal)
       - name: MAS_APP_ID
         value: $(params.mas_app_id)
+      - name: OCP_SERVER
```
Comment (andrercm) [inline tekton/src/tasks/gitops/gitops-deprovision-app-config.yml.j2:66]: same as above...
why we're using `$(params.cluster_url)` to set `OCP_SERVER`?
should this be setting `CLUSTER_URL` or the other way around?

File: tekton/src/tasks/gitops/gitops-deprovision-app-install.yml.j2 [hunk 3]
```diff
@@ -56,10 +59,18 @@ spec:
         value: $(params.force_removal)
       - name: MAS_APP_ID
         value: $(params.mas_app_id)
+      - name: OCP_SERVER
```
Comment (andrercm) [inline tekton/src/tasks/gitops/gitops-deprovision-app-install.yml.j2:62]: same as above:
why we're using `$(params.cluster_url)` to set `OCP_SERVER`?
should this be setting `CLUSTER_URL` or the other way around?

**Review by andrercm — CHANGES_REQUESTED** [review]
Just a quick question..

File: tekton/src/tasks/gitops/gitops-delete-kafka-config.yml.j2 (hunk 1 above)
Comment (padmankosalaram) [reply tekton/src/tasks/gitops/gitops-delete-kafka-config.yml.j2:62]: value set in param cluster_url is set to env variable OCP_SERVER

**Review by andrercm — APPROVED** [review]
Regarding my questions, Padmanabhan said ok we'll be reusing the value already set from cluster_url to also set OCP_SERVER.
Approving this PR.

File: tekton/src/tasks/gitops/gitops-deprovision-app-config.yml.j2 (hunk 2 above)
Comment (padmankosalaram) [reply tekton/src/tasks/gitops/gitops-deprovision-app-config.yml.j2:66]: same as above. The value set in param cluster_url is set to env variable OCP_SERVER

**Pull request merged**